core/templates/core/login.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Logowanie{% endblock %}
+
+{% block content %}
+<script type="module">
+    const canvas = document.getElementById('starback-bg');
+          const starback = new Starback(canvas, {
+              type: 'dot',
+              quantity: 250,
+              height: window.innerHeight,
+              width: window.innerWidth,
+              backgroundColor: [],
+              randomOpacity: true,
+          });
+          function resizeCanvas() {
+              canvas.width = window.innerWidth;
+              canvas.height = window.innerHeight;
+          }
+          window.addEventListener('resize', resizeCanvas);
+</script>
```

Comment:
move this script to block head

Reply:
I don't know how even with a help of Norbiros so I won't do it work